### PR TITLE
Add English WNP for Firefox 105 [fix #12130]

### LINKED ISSUE #12130: WNP 105 for North America
## Description

Launch Date: Sept 20th

Design & Copy: [here](https://www.figma.com/file/CZvzHgU9NQMQUm34lomIl5/Firefox-WNP?node-id=2234%3A40182)

Target: English

### PR DESCRIPTION
**Pull request by craigcook**
## One-line summary

Adds English whatsnew page for Firefox 105.

## Issue / Bugzilla link

#12130 

## Testing

http://localhost:8000/en-US/firefox/105.0/whatsnew/

### CODE REVIEW

**Comment by craigcook** [comment]
Note this is likely to cause some merge conflicts with https://github.com/mozilla/bedrock/pull/12090 so whichever one merges first, the other one will need a careful rebase.

**Comment by alexgibson** [comment]
I just merged https://github.com/mozilla/bedrock/pull/12090 (sorry! 🙈) so this will need a rebase. Doesn't look too bad thankfully :)

**Comment by alexgibson** [comment]
I'll review this whilst I'm here by way of an apology

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx105-en.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+</section>
+
+<section class="wnp-content-extra">
+  <div class="mzp-l-content mzp-t-content-lg">
+
+    <h3 class="mzp-c-section-heading">Some topics you’ll cover:</h3>
+
+    <div class="mzp-l-columns mzp-t-columns-three mzp-t-picto-center">
+      <div class="mzp-c-picto">
+        <img src="{{ static('img/firefox/whatsnew/whatsnew105/privacy.png') }}" width="320" height="" alt="216" class="mzp-c-picto-image">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx105-en.html:41]: NIt: looks like you need to swap the `height` and `alt` values around on these three images:
```suggestion
        <img src="{{ static('img/firefox/whatsnew/whatsnew105/privacy.png') }}" width="320" height="216" alt="" class="mzp-c-picto-image">
```

File: media/css/firefox/whatsnew/whatsnew-105-en.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    padding-top: $spacing-md;
+    text-align: center;
+}
+
+.wnp-main-title {
+    @include text-title-xl;
+}
+
+.wnp-main-img {
+    max-width: $content-md;
```
Comment (alexgibson) [inline media/css/firefox/whatsnew/whatsnew-105-en.scss:26]: Issue (non-blocking): looks like this is preventing the image from resizing on smaller viewports?

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx105-en.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  {{ css_bundle('firefox_whatsnew_105_en') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+<section class="wnp-content-main mzp-l-content mzp-t-content-lg">
+  <h2 class="wnp-main-title">How to talk to your kids about online safety</h2>
+
+  <img src="{{ static('img/firefox/whatsnew/whatsnew105/tech-talk.png') }}" width="720" height="405" alt="The Tech Talk" class="wnp-main-img">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx105-en.html:23]: Suggestion (non-blocking): a responsive image would be a nice to have here, mainly so we're not serving people on low res monitors a @2x sized image?

**Review by alexgibson — CHANGES_REQUESTED** [review]
A couple of tiny issues / nits, but otherwise this all looks good (after a rebase and minor shuffle of the unit tests).

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx105-en.html (hunk 1 above)
Comment (craigcook) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx105-en.html:41]: 🤦

**Comment by craigcook** [comment]
Rebased and updated. I initially skipped responsive images because the high-res ones were already such small files and I don't know how many people still have low-res screens these days (other than me, because I haven't bought a new monitor in 15 years). But the new low-res images are even tinier still and I guess every little performance boost helps.

**Review by alexgibson — APPROVED** [review]
r+

**Pull request merged**